### DISCUSSION TITLE
Let's find a cool name for the library!

### DISCUSSION THREAD

**Discussion by Stakhan** [CCInc/3d-ml#38] [General]
The name of this new library is not written in stone, and we would like to involve everybody in the selection process!

Ideally, the new name should include the following (in a way or another):
+ 3d data
+ point clouds
+ machine learning
+ pytorch

Just add your name proposals below and we will directly up-vote or down-vote them in the comment section!

**Comment by Stakhan** [comment]
pypoints3d

**Comment by Stakhan** [comment]
torch3d

**Reply by Stakhan** [reply]
Too close to https://pytorch3d.org/

**Comment by Stakhan** [comment]
3dtorch

**Reply by Stakhan** [reply]
Name too close to https://pytorch3d.org/

**Comment by leo-stan** [comment]
py3Dml

**Comment by leo-stan** [comment]
3d-ml

**Reply by Stakhan** [reply]
Wouldn't this sound too close to https://github.com/isl-org/Open3D-ML ?

**Comment by leo-stan** [comment]
3d-dl :smile:

**Comment by aaronfderybel** [comment]
**F**lexibe **R**eproducible **3**d point cloud models with **py**torch
FR3Dpy ?

**Reply by Stakhan** [reply]
Putting the words "flexible" and "reproducible" is good.
But FR3Dpy doesn't convince me that much. "FR" is the ISO country code for France, which is a bit confusing.
Maybe rfml3d ? "**r**eproducible and **f**lexible **m**achine **l**earning on **3d** point clouds?

**Comment by MCvin** [comment]
pytorch-pointclouds
